int64 ticks_fl_prev
int64 ticks_fr_prev
int64 ticks_rl_prev
int64 ticks_rr_prev
---
        